modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaBaseModelListenerCheck.java

```diff
@@ -38,34 +41,73 @@ protected String format(
 			return content;
 		}
 
+		_extendedClassType = _getExtendedClassType(content);
+
 		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()) {
 			if (!childJavaTerm.isJavaMethod()) {
 				continue;
 			}
 
 			JavaMethod javaMethod = (JavaMethod)childJavaTerm;
 
-			String newJavaMethodContent = _formatMethodDefinition(javaMethod);
+			String newJavaContent = _formatMethod(javaMethod);
 
 			content = StringUtil.replace(
-				content, javaMethod.getContent(), newJavaMethodContent);
+				content, javaMethod.getContent(), newJavaContent);
 		}
 
 		return content;
 	}
 
-	private String _formatMethodDefinition(JavaMethod javaMethod) {
-		String javaMethodContent = javaMethod.getContent();
-
+	private String _formatMethod(JavaMethod javaMethod) {
 		String javaMethodName = javaMethod.getName();
 
-		if (!javaMethodName.equals("onAfterUpdate") &&
-			!javaMethodName.equals("onBeforeUpdate")) {
+		if (javaMethodName.equals("onAfterUpdate") ||
+			javaMethodName.equals("onBeforeUpdate")) {
+
+			return _formatMethodDefinitionWithParameterUpgrade(javaMethod);
+		}
+
+		return _formatMethodDefinitionWithoutParameterUpgrade(javaMethod);
+	}
 
+	private String _formatMethodDefinitionWithoutParameterUpgrade(
+		JavaMethod javaMethod) {
+
+		String javaMethodContent = javaMethod.getContent();
+
+		Matcher matcher = _superMethodPattern.matcher(javaMethodContent);
+
+		if (!matcher.find()) {
 			return javaMethodContent;
 		}
 
-		javaMethodContent = _formatSuper(javaMethodContent);
+		String methodCall = JavaSourceUtil.getMethodCall(
+			javaMethodContent, matcher.start());
+
+		methodCall = methodCall.trim();
+
+		String parameterName = JavaSourceUtil.getParameters(methodCall);
```

Comment:
I don't know about this variable name. 
I understand that the regex will only match single parameter calls, so, it will be a unique `parameterName` in the return of this method. But one plural method returns saved as a singular variable name it's weird.

Another possibility would be `parameterNamesString`. But, let's keep `parameterName` and see if it's okay in the final review.

Reply:
Okay we can discuss it further in final review, I just thought it would help imply it can technically only return singular, even though the getParameters method can return any amount of parameters present